service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftOrdersApprovedEventHandler.java

```diff
@@ -201,6 +266,10 @@ public void sendNotificationToDigitalRepresentatives(final DraftOrdersApproved e
     public void sendNotificationToEmailRepresentatives(final DraftOrdersApproved event) {
         CaseData caseData = event.getCaseData();
         List<HearingOrder> approvedOrders = event.getApprovedOrders();
+        if (event.getApprovedOrders().isEmpty()) {
+            log.info("No non-confidential approved orders. skip sendDocumentToPostRecipients");
```

Comment:
Change to `sendDocumentToPostRecipients`

Reply:
nice spotting, updated